projects/orquestra-sdk/docs/tutorials/installing-macos-linux.rst

```diff
@@ -18,6 +18,17 @@ Install the Orquestra Workflow SDK by running:
 This will install ``orquestra-sdk`` and all of its dependencies.
 Note that ``orquestra-sdk`` also includes the :doc:`Orquestra command-line interface <../quickref/cli-reference>`.
 
+Note:
+
+.. code-block:: rst
+
+    When running the command above, you may observe an issue with it trying to install both 0.64.0 and 0.63.0
```

Comment:
Suggestion: This line of document will become very outdated rather quickly (even now we have 0.65 already).

If I might suggest: I wouldn't focus on specific versions, I'd rather see a helpful non-version specific tips like:
"dependency conflicts error of 'ray==X.Y.Z; extra == "all" might suggest problems with Python versions. Right Now its best to have your environment at Python version 3.11.X (you can check your Python version using `python --version`

Also great comment about virtual environment! Thats very useful as well